src/current/cockroachcloud/upgrade-to-v24.1.md

```diff
@@ -134,7 +134,9 @@ Use the [DB Console]({% link cockroachcloud/tools-page.md %}) or your own toolin
 
 ### Expect temporary limitations
 
-Most {{ page.page_version }} features can be used right away, but some will be enabled only after the upgrade has been finalized. Attempting to use these features before finalization will result in errors. {% comment %}TODO: List of temporary limitations for GA{% endcomment %}
+Most {{ page.page_version }} features can be used right away, but some will be enabled only after the upgrade has been finalized. Attempting to use these features before finalization will result in errors.
+
+- Splits no longer hold latches for time proportional to the range size while computing MVCC statistics. Instead, MVCC statistics are pre-computed before the critical section of the split. As a side effect, the resulting statistics are no longer 100% accurate because they may correctly distribute writes concurrent with the split. To mitigate against this potential inaccuracy, and to prevent the statistics from drifting after successive splits, the existing stored statistics are re-computed and corrected if needed during the non-critical section of the split. [#119894](https://github.com/cockroachdb/cockroach/pull/119894)
```

Comment:
'splits' can link to https://www.cockroachlabs.com/docs/v24.1/architecture/distribution-layer.html#range-splits

'latches' can link to https://www.cockroachlabs.com/docs/v24.1/architecture/transaction-layer#latch-manager

'mvcc' can link to https://www.cockroachlabs.com/docs/v23.2/architecture/storage-layer.html#mvcc 

this is duplicated with the other page and should be an include